packages/astro/package.json

```diff
@@ -17,7 +17,7 @@
     "./snowpack-plugin-jsx": "./snowpack-plugin-jsx.cjs",
     "./components": "./components/index.js",
     "./debug": "./components/Debug.astro",
-    "./components/*": "./components/*",
```

Comment:
Why make this change? I'm unsure of what this changes - I guess you can no longer do `import Markdown from "astro/components/Markdown.astro";`?

Reply:
That's correct, all are expected to now go through `import {} from 'astro/components';`

Was that ever documented? I thought that was only needed internally for the Prism component (which is why it is still added here).

Reply:
ah, I think that was only for `Debug`, no?

Reply:
This was in the export map before we supported reexporting Astro components.

Reply:
ah, maybe we just keep it then and remove in v1.0

Reply:
added back